cpp/test/handle.cpp

```diff
@@ -64,18 +62,6 @@ TEST(Raft, GetHandleFromPool) {
   ASSERT_EQ(parent.get_device(), child.get_device());
 }
 
-TEST(Raft, GetHandleFromPoolPerf) {
```

Comment:
I had to remove this test because for some reason, the destructor for `parent` was being called while in the for loop. Since this is just a temporary fix, and my handle refactor will take care of the copy constructor for `raft::handle_t`, we should be fine